ADOPTERS.md

```diff
@@ -222,3 +222,5 @@ _You can do this by using the [Adopter form](https://info.backstage.spotify.com/
 | [MSCI](http://msci.com)                                            | [Stephen Burrows](mailto:stephen.burrows@msci.com) | Developer portal, service catalog, documentation and tooling
 | [ESW](https://esw.com)                                            | [Alisson Fabiano](https://github.com/afabianoo), [Bruno Quintella](https://github.com/quintelab) and [Guilherme Oenning](https://github.com/goenning) | Backstage is our one stop shop to find everything related to all our services, such as ownership, dependencies, production status, tech health and much more.
 | [FanDuel](https://fanduel.com)                                            | [Diego Herrera](https://github.com/diegoh), [Christy Campbell](https://github.com/FD-ChristopherCampbell) | We use backstage as our developer portal to provide visibility of our software, ownership, strategy, and the state of maturity across disciplines.
+| [Affinidi](https://affinidi.com) | [Dmytro Shamenko](https://github.com/idestis), [Denis Fastovets](https://www.linkedin.com/in/denis-fastovets/) | The Backstage is used 
+in the company to enhance development acceleration inside of the company and helps us keep the house in order, from cost efficiency and up to accountability.
```

Comment:
Almost there! You have a enter here which makes these last lines fall in their own table cell. So, if you do some backspaces then we should be fine 😄